### PR TITLE
React native sdk

### PR DESCRIPTION
**Pull request by philolo1**
This creates the first version for the react-native-sdk for proton.

### CODE REVIEW

File: packages/proton-react-native-sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        from: rootStore.actor,
+        to: protonSDK.requestAccount,
+        quantity: '5.000000 XUSDT',
+        memo: 'Taskly',
+      },
+    },
+  ];
+  const tx = await protonSDK.sendTransaction(actions);
+
+  // navigate to the subscriped page
```
Comment (joannerd) [inline packages/proton-react-native-sdk/README.md:144]: Small typo: `subscriped` -> `subscribed`

File: packages/proton-react-native-sdk/test/blah.test.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { sum } from '../src';
+
+describe('blah', () => {
```
Comment (joannerd) [inline packages/proton-react-native-sdk/test/blah.test.ts:3]: Do we need this demo test?

File: packages/proton-react-native-sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    } catch (e) {
+      return e;
+    }
+  };
+}
+
+// Usage
+const protonSDK = new ProtonSDK();
+
+// usage login()
```
Comment (joannerd) [inline packages/proton-react-native-sdk/README.md:107]: Can we separate the `login`, `sendTransaction`, `navigate`, `logout`, and `restoreSession` examples into different code snippets with the method name as a header before the usage example (kind of like on [docs.protonchain.com](https://docs.protonchain.com/sdk/proton-web-sdk#login))?

File: packages/proton-react-native-sdk/package.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    "lint": "tsdx lint",
+    "prepare": "tsdx build",
+    "size": "size-limit",
+    "analyze": "size-limit --why"
+  },
+  "dependencies": {
+    "@protonprotocol/proton-browser-transport": "^2.5.4",
+    "@protonprotocol/proton-link": "^2.5.8",
+    "@protonprotocol/proton-signing-request": "^2.5.4",
+    "@protonprotocol/protonjs": "^21.0.21",
```
Comment (cindyssun5) [inline packages/proton-react-native-sdk/package.json:32]: Shouldn't this be pulling packages from @proton/browser-transport, @proton/link, @proton/signing-request, and @proton/js? The new packages have been published in the proton npm org.

File: packages/proton-react-native-sdk/README.md (hunk 1 above)
Comment (philolo1) [reply packages/proton-react-native-sdk/README.md:144]: done thanks

File: packages/proton-react-native-sdk/test/blah.test.ts (hunk 2 above)
Comment (philolo1) [reply packages/proton-react-native-sdk/test/blah.test.ts:3]: No, lets delete this.

File: packages/proton-react-native-sdk/src/index.ts [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+  // Add chain ID if not present
+  if (!linkOptions.chainId) {
+    const info = await linkOptions.rpc!.get_info();
+    linkOptions.chainId = info.chain_id;
+  }
+
+  // Add storage if not present
+  console.log('storage');
+  console.log('storage 2');
```
[the remark this answers is not in the document]
Comment (philolo1) [reply packages/proton-react-native-sdk/src/index.ts]: fixed

File: packages/proton-react-native-sdk/package.json (hunk 4 above)
Comment (philolo1) [reply packages/proton-react-native-sdk/package.json]: You are right, thank you!

File: packages/proton-react-native-sdk/README.md (hunk 3 above)
Comment (philolo1) [reply packages/proton-react-native-sdk/README.md:107]: Sure will do

**Pull request merged**